src/solvers/mp/TreeManager.h

```diff
@@ -72,18 +72,18 @@ struct ShiftNode: public MyNode{
 	const LiveNurse* pNurse_;
 	const int day_;
 	const bool work_;
-	vector<int> forbiddenShifts_;
+    std::vector<int> forbiddenShifts_;
 };
 
 // Node that corresponds to branching on a penalized original variable
 // The variable can relate to cover constraints, total shifts or total week-ends
 //
 struct PenaltyNode: public MyNode{
-	PenaltyNode(int index, MyNode* pParent, LiveNurse* pNurse, int day, bool work, vector<int>& forbiddenShifts):
+	PenaltyNode(int index, MyNode* pParent, LiveNurse* pNurse, int day, bool work, std::vector<int>& forbiddenShifts):
```

Comment:
Do we still use this branching rule? It seems that the function branchOnPenalty in TreeManager.cpp does not even do anything (the input reference to a candidate is never modified). Delete? Store in an issue on research of better branching rules?

Reply:
never used, but I do not remember to have coded it, so I have just left it there !

Reply:
deleted in 6235f15156914486d2042e7e493a0bf2a28d18aa